src/components/reactTasks/comment.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+export default class Comment extends React.Component{
+
+  constructor(){
+    super()
+    this.state = {comments: ['Brilliant!', 'Terrible'], value: ''}
+  }
+
+  handleChange(value) {
+    console.log(value)
+    this.setState({value: value});
```

Comment:
If you're not using `;` you have to be consistent throughout. So remove this one.